src/main/resources/resources/logisim/en/fsm.properties

```diff
@@ -0,0 +1,9 @@
+#
+# This file contains all messages of the FSM component
```

Comment:
It would be good to add translated versions of this file at least to the French and German internationalisation directories.